test/credentials.spec.js

```diff
@@ -1479,22 +1479,37 @@
           });
         });
       });
-      return it('should refresh expired master credentials when refreshing self', function() {
+      it('should refresh expired master credentials when refreshing self', function() {
         var masterCreds, refreshSpy;
         masterCreds = new AWS.Credentials('akid', 'secret');
         masterCreds.expired = true;
         refreshSpy = helpers.spyOn(masterCreds, 'refresh');
         creds = new AWS.TemporaryCredentials({
           RoleArn: 'ARN'
         }, masterCreds);
-        creds.createClients();
```

Comment:
I removed this because it appears unnecessary since `refresh` will call `createClients`.